core/templates/dev/head/components/answer_group_editor.html

```diff
@@ -79,7 +79,7 @@
     <div class="oppia-rule-save-cancel-buttons">
       <div class="pull-right">
         <button type="button" class="btn btn-default" ng-click="cancelThisFeedbackEdit()">Cancel</button>
-        <button type="button" class="btn btn-success protractor-test-save-outcome-feedback" ng-disabled="editAnswerGroupForm.editFeedbackForm.$invalid" ng-click="saveThisFeedback()">Save Feedback</button>
+        <button type="button" class="btn btn-success protractor-test-save-outcome-feedback" ng-click="saveThisFeedback()">Save Feedback</button>
```

Comment:
I added this in the first commit .. I thought it is not needed at all. 
I have added back. Done.